src/services/SendEmailProcessor.ts

```diff
@@ -44,15 +49,62 @@ export class SendEmailProcessor {
 
 	async processRequest(eventBody: any): Promise<EmailResponse> {
 		const email = Email.parseRequest(JSON.stringify(eventBody.Message));
```

Comment:
Could we rename `email` to `message` or something? Reading `email.userId` below confused me!

Reply:
I think it would be rather better to rename the Email class here to User as the attributes in there are more related to a user details and can then rename the const to user instead. Any thoughts?